packages/docusaurus-theme-classic/src/theme/SiteMetadata/index.tsx

```diff
@@ -33,16 +33,22 @@ function AlternateLangHeaders(): JSX.Element {
   return (
     <Head>
       {Object.entries(localeConfigs).map(([locale, {htmlLang}]) => (
-        <link
-          key={locale}
-          rel="alternate"
-          href={alternatePageUtils.createUrl({
-            locale,
-            fullyQualified: true,
-          })}
-          hrefLang={htmlLang}
-        />
+		    <>
+		      <link
+            key={locale}
+            rel="alternate"
+            href={alternatePageUtils.createUrl({
+              locale,
+              fullyQualified: true,
+            })}
+            hrefLang={htmlLang}
+          />
+		      <meta property="og:locale:alternate" content={locale} />
+		    </>
       ))}
+
+	    <meta property="og:locale" content={defaultLocale} />
```

Comment:
We should use the "current locale" (and not the default locale) as bcp47 code (`htmlLang` attribute like in hreflang links), but converted to og locale format (see https://github.com/facebook/docusaurus/issues/8887#issuecomment-1514981713)